scantools/run_rendering.py

```diff
@@ -26,17 +26,23 @@ def run(capture: Capture, session_id: str, mesh_id: str = 'mesh',
     depths = session.depths
     if depths is None:
         depths = session.depths = RecordsDepth()
-    output_dir = capture.data_path(session_id)
+    output_dir = capture.proc_path(session_id)
```

Comment:
I think this breaks the pseudo-GT pipeline which might try to read the depth-maps according to depths.txt from data_path instead of proc_papt.

Reply:
You are right, the `depths.txt` paths are relative to the `raw_data/` folder. This is actually a problem; we are adding generated data in the `raw_data/` folder. It might be confusing but for the moment I will revert it (I will push a commit soon).

Reply:
Yes, having the rendered depth in `proc/` would make more sense, but then it can't be treated identically to sensor depth - we'd need a mechanism to add a depth reader to `Proc`.